pkg/ingress/controller/controller.go

```diff
@@ -108,6 +108,18 @@ const (
 	// IngressControllerTag is added to the related resources.
 	IngressControllerTag = "octavia.ingress.kubernetes.io"
 
+	// IngressAnnotationTimeoutClientData is the timeout for client data in ms
+	IngressAnnotationTimeoutClientData = "octavia.ingress.kubernetes.io/timeout-client-data"
+
+	// IngressAnnotationTimeoutClientData is the timeout for member data in ms
+	IngressAnnotationTimeoutMemberData = "octavia.ingress.kubernetes.io/timeout-member-data"
+
+	// IngressAnnotationTimeoutMemberConnet is the timeout for memer connect in ms
+	IngressAnnotationTimeoutMemberConnect = "octavia.ingress.kubernetes.io/timeout-member-connect"
+
+	// IngressAnnotationTimeoutMemberConnet is the timeout for memer connect in ms
```

Comment:
Maybe for completeness you could mention what's the default value (I suppose it's an Octavia setting?)

Reply:
Should I just document default values here?

Reply:
It's not an octavia config argument but it's available for LB, when the controller starts to create listener it doesn't pass this value post so that LoadBalancer default values can get applied, so basically we are passing this in CreateOpts to let override the defaults available in LB. I will still add the docs of the CLI configs.

Reply:
fixed